cli/cmd/diagnose.go

```diff
@@ -7,18 +7,70 @@ import (
 	"fmt"
 	"github.com/odigos-io/odigos/cli/cmd/resources"
 	"github.com/odigos-io/odigos/cli/pkg/kube"
+	"github.com/odigos-io/odigos/k8sutils/pkg/client"
 	"github.com/spf13/cobra"
 	"io"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
 	"os"
 	"path/filepath"
+	"sigs.k8s.io/yaml"
 	"sync"
 	"time"
 )
 
 const (
-	logBufferSize = 1024 * 1024 // 1MB buffer size for reading logs in chunks
+	logBufferSize   = 1024 * 1024 // 1MB buffer size for reading logs in chunks
+	LogsDir         = "Logs"
+	CRDsDir         = "CRDs"
+	CRDName         = "crdName"
+	CRDGroup        = "crdGroup"
+	actionGroupName = "actions.odigos.io"
+	odigosGroupName = "odigos.io"
+)
+
+var (
+	diagnoseDirs = []string{LogsDir, CRDsDir}
+	CRDsList     = []map[string]string{
+		{
+			CRDName:  "addclusterinfos",
+			CRDGroup: actionGroupName,
+		},
```

Comment:
nit: since they all look and are being accessed the same way, you can make it into a struct which is more idomatic IMO and easier to read